src/msg/Message.h

```diff
@@ -187,8 +187,7 @@
 #define MSG_CRC_ALL            (MSG_CRC_DATA | MSG_CRC_HEADER)
 
 // Xio Testing
-#define MSG_DATA_PING		  0x602
-
+//#define MSG_DATA_PING		  0x602
```

Comment:
we can recycle this message id after two releases, right? but, yeah, it's not a big deal .